cli-plugins/manager/manager_test.go

```diff
@@ -92,10 +92,14 @@ func TestErrPluginNotFound(t *testing.T) {
 func TestGetPluginDirs(t *testing.T) {
 	cli := test.NewFakeCli(nil)
 
-	expected := []string{config.Path("cli-plugins")}
+	pluginDir, err := config.Path("cli-plugins")
+	assert.NilError(t, err)
+	expected := []string{pluginDir}
```

Comment:
nit: could be combined to a single line


```suggestion
	expected := append([]string{pluginDir}, defaultSystemPluginDirs...)
```